archive/scripts/vote_sr_v2_holesky.py

```diff
@@ -358,8 +358,8 @@ def start_vote(tx_params: Dict[str, str], silent: bool) -> Tuple[int, Optional[T
             agent_forward(
                 [
                     (
-                        contracts.csmHashConsensus.address,
-                        contracts.csmHashConsensus.updateInitialEpoch.encode_input(CS_ORACLE_INITIAL_EPOCH),
+                        contracts.csm_hash_consensus.address,
```

Comment:
Not sure that we need support archived scripts, as far as I know they mostly kept for history only.